app/assets/templates/vocabs/vocabs.html.erb

```diff
@@ -0,0 +1,92 @@
+<nav class="navbar navbar-default">
```

Comment:
you can extract the navbar into a separate directive:

- Define navbar directive: https://github.com/WDI-HK-8/Formidable-web/blob/master/app%2Fassets%2Fjavascripts%2Fdirectives%2Fnavbar.js.erb
- Define navbar template: https://github.com/WDI-HK-8/Formidable-web/blob/master/app%2Fassets%2Ftemplates%2Fnav%2Fnavbar.html

And then you can use it in any view by simply putting `<navbar></navbar>` :)